.github/workflows/ci.yml

```diff
@@ -573,6 +573,57 @@ jobs:
       - name: test-cri
         run: sudo BOX=$BOX vagrant up --provision-with=selinux,install-runc,install-gotestsum,test-cri
 
+  integration-vagrant-v1:
```

Comment:
WDYT of adding it to the above integration-vagrant itself, using http://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#expanding-or-adding-matrix-configurations for `io.containerd.runc.v1,  io.containerd.runtime.v1.linux` so we can test v1 shim only on almalinux/8